### PR TITLE
VV-331 Modal popups not working

### PR DESCRIPTION
**Pull request by rdgfuentes**
https://globalfishingwatch.atlassian.net/browse/VV-331
- [x] Fixed redirect tp load resources when app is on subdomain and also in path (prod)
- [x] Fixed app title and base urls
- [x] Fixed modal popups not appearing

### CODE REVIEW

File: apps/vessel-history/pages/_document.tsx [hunk 1]
```diff
@@ -201,12 +190,13 @@ class MyDocument extends Document {
           </head>
         </Head>
         <body>
+          <div id="root"></div>
```
Comment (j8seangel) [inline apps/vessel-history/pages/_document.tsx:193]: I think we can avoid having it here by these two options:
- [use the appSelector prop](https://github.com/GlobalFishingWatch/frontend/blob/develop/apps/fishing-map/features/app/App.tsx#L229) in the Modal ui-component
- update [ui-component default value](https://github.com/GlobalFishingWatch/frontend/blob/develop/libs/ui-components/src/modal/Modal.tsx#L29) as CRA is not used anymore (or keep it as it is and add another fallback value for next apps)

what do you prefer?

File: apps/vessel-history/nginx.conf [hunk 2]
```diff
@@ -31,7 +31,7 @@ http {
         listen ${PORT};
         root /usr/share/nginx/www/;
 
-        location ~* ^/map/(.+\..+)$ {
+        location ~* ^/vessel-viewer/(.+\..+)$ {
```
Comment (j8seangel) [inline apps/vessel-history/nginx.conf:34]: ouch, sorry for this! 
my fault copying and pasting from fishing-map

File: apps/vessel-history/pages/_document.tsx (hunk 1 above)
Comment (rdgfuentes) [reply apps/vessel-history/pages/_document.tsx]: Good catch, thanks! I'll use the `appSelector="__next"` in the Modal ui-component

File: apps/vessel-history/nginx.conf (hunk 2 above)
Comment (rdgfuentes) [reply apps/vessel-history/nginx.conf:34]: No problem 👍

**Pull request merged**